test/parallel/test-stream-writable-destroy.js

```diff
@@ -354,7 +354,7 @@ const assert = require('assert');
     assert.strictEqual(err.message, 'asd');
   }));
   write.end('asd', common.mustCall((err) => {
-    assert.strictEqual(err.message, 'asd');
+    assert.strictEqual(err.code, 'ERR_STREAM_DESTROYED');
```

Comment:
I'm a bit puzzled by this change. Why is this needed/necessary?

Reply:
That's how write works in relation to `_write`, i.e. we no longer listen to **every** error, just to errors directly related to `end()`/`_finish` everything else becomes this error.